torchao/quantization/autoquant.py

```diff
@@ -15,14 +15,19 @@
 from .quant_primitives import (
     safe_int_mm,
 )
-from torchao.utils import TORCH_VERSION_AFTER_2_3
+from torchao.utils import TORCH_VERSION_AFTER_2_3, TORCH_VERSION_AFTER_2_5
 from torchao.quantization.utils import quantize_activation_per_token_absmax
 
 import torch.nn.functional as F
+
 try:
     from torch._inductor.utils import do_bench
-except:
-    from torch._inductor.runtime.runtime_utils import do_bench
+except ImportError:
+    try:
+        from torch._inductor.runtime.runtime_utils import do_bench
+    except ImportError:
+        from torch._inductor.runtime.benchmarking import benchmarker
+        do_bench = benchmarker.benchmark
```

Comment:
Do we need these imports?

Reply:
AO needs to work over multiple pytorch versions and this api has been broken multiple times

We can explore getting rid of the usage completely @HDCharles

Reply:
Can we not inline the imports in the IF/ELSE blocks later on when they get used (I see some already are?)?